### ISSUE TITLE
Blog about archivist.github

### ISSUE THREAD

**Issue by pbiecek** [pbiecek/archivist#214]
After reaching CRAN

**Comment by MarcinKosinski** [comment]
Sure we will :)
But I'll send it to CRAN only after we'll clean archivist repository and functionalities.
Additionally I though you'd like to blog first about other functionalities?

**Comment by pbiecek** [comment]
Right, let's start with simpler things (like #213)

2016-02-08 11:26 GMT+01:00 Marcin Kosiński <notifications@github.com>:

> Sure we will :)
> But I'll send it to CRAN only after we'll clean archivist repository and
> functionalities.
> Additionally I though you'd like to blog first about other functionalities?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbiecek/archivist/issues/214#issuecomment-181295954>.
>



-- 
pozdrawiam serdecznie,
Przemysław Biecek

**Comment by MarcinKosinski** [comment]
TODO

- review of archivist.github
- upload archivist.github to CRAN
- short website/vignette to a blog http://marcinkosinski.github.io/archivist.github/ + add information to DESCRIPTION and man pages about this webpage
- figure out how one can add html as a vignette (can't pass .Rmd file because it requires to load my password to github)

**Comment by MarcinKosinski** [comment]
Statis HTML vignettes: https://cran.r-project.org/web/packages/R.rsp/vignettes/R_packages-Static_PDF_and_HTML_vignettes.pdf

**Comment by MarcinKosinski** [comment]
https://github.com/MarcinKosinski/archivist.github/issues

**Comment by MarcinKosinski** [comment]
@pbiecek  What would you say about such blog post :)? R Heroes are very popular recently :) ![archivist_github](https://cloud.githubusercontent.com/assets/6773454/13982817/708b8370-f0ec-11e5-8e71-aba57536e59e.jpg)

**Comment by pbiecek** [comment]
It's super cool!



Dnia 23.03.2016 o godz. 11:43 Marcin Kosiński <notifications@github.com> napisał(a):

**Comment by MarcinKosinski** [comment]
We'll change `forced` to `faced`

**Comment by MarcinKosinski** [comment]
Fixed version without typos
https://raw.githubusercontent.com/MarcinKosinski/archivist.github/master/scripts/archivist_rhero.png

**Comment by pbiecek** [comment]
Shouldn't it be 'in Backup City'
like here:
http://english.stackexchange.com/questions/107977/which-one-is-correct-at-the-city-or-in-the-city


2016-03-23 12:38 GMT+01:00 Marcin Kosiński <notifications@github.com>:

> Fixed version without typos
>
> https://raw.githubusercontent.com/MarcinKosinski/archivist.github/master/scripts/archivist_rhero.png
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pbiecek/archivist/issues/214#issuecomment-200308647>
>



-- 
pozdrawiam serdecznie,
Przemysław Biecek

**Comment by MarcinKosinski** [comment]
Thanks. Fixed this. I'll blog about archivist.github during upcoming week

**Comment by MarcinKosinski** [comment]
I was still waiting for Tal Galili to respond for my request to join R-bloggers with my pet blog. After that I was about to be happy to post about archivist.github. 

But I think there is also another option to feature on R-bloggers. A good idea is to write on r-packages-devel mailing list, where most new messages are re-posted automatically on R-bloggers, like this announcement http://www.r-bloggers.com/rzabbix-easy-and-direct-communication-with-zabbix-api/

**Comment by pbiecek** [comment]
@MarcinKosinski now, when your blog is connected to RBloggers you could blog about archivist.github. Feel free to mention that archivist will be presented on both useR and eRum. Here is the link for useR: http://schedule.user2016.org/event/7BYx/how-to-use-the-archivist-package-to-boost-reproducibility-of-your-research

**Comment by MarcinKosinski** [comment]
I remember about that but first I wanted to finish Graphs for Beta and Bit, then I wanted to close 11 issues at archivist.github and at the end I wanted to post a quick overview Bout archivist.github before eRum :) in this week it is hard, as I am Spending most of the time with my Family but I am planning to finish this till the end of the week

Marcin Kosinski

Dnia 01.06.2016 o godz. 21:56 Przemysław Biecek <notifications@github.com> napisał(a):

**Comment by pbiecek** [comment]
No pressure ;-)
I need to prepare one by my own (about knitr+archivist)

2016-06-01 23:59 GMT+02:00 Marcin Kosiński <notifications@github.com>:

> I remember about that but first I wanted to finish Graphs for Beta and
> Bit, then I wanted to close 11 issues at archivist.github and at the end I
> wanted to post a quick overview Bout archivist.github before eRum :) in
> this week it is hard, as I am Spending most of the time with my Family but
> I am planning to finish this till the end of the week
>
> Marcin Kosinski
>
> Dnia 01.06.2016 o godz. 21:56 Przemysław Biecek <notifications@github.com>
> napisał(a):
>
> > @MarcinKosinski now, when your blog is connected to RBloggers you could
> blog about archivist.github. Feel free to mention that archivist will be
> presented on both useR and eRum. Here is the link for useR:
> http://schedule.user2016.org/event/7BYx/how-to-use-the-archivist-package-to-boost-reproducibility-of-your-research
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub, or mute the thread.
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pbiecek/archivist/issues/214#issuecomment-223137810>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEaPvqf2MfPmxWNbUFKQXiI4bUGROi6bks5qHgCtgaJpZM4HVZj_>
> .
>



-- 
pozdrawiam serdecznie,
Przemysław Biecek

**Comment by MarcinKosinski** [comment]
This will be mine next blog-post . It's on the top of the ToDo's :)

Marcin Kosinski

Dnia 02.06.2016 o godz. 00:01 Przemysław Biecek <notifications@github.com> napisał(a):

**Comment by MarcinKosinski** [comment]
Moved to https://github.com/MarcinKosinski/archivist.github/issues/20

**Comment by MarcinKosinski** [comment]
I have posted this yesterday at 18:30 polish time.
AfterI have posted this post, 4 more posts appeared on r-bloggers with date 13-06-2016.
Today I have entered r-bloggers and this blog is 9/10th from the bottom and have date 12-06-2016 :/

So it wasn't probably seen on main placement on the top.. I wonder if it's even going to be send by e-mail to r-bloggers lists. I have no idea why it was published so late and with different date, and why it didn't appear on top.

But it's on r-bloggers anyway

http://www.r-bloggers.com/r-hero-saves-backup-city-with-archivist-and-github/
